cytoscape-panzoom.js

```diff
@@ -407,13 +432,52 @@ SOFTWARE.
             return false;
           });
 
+          sliding = false;
+          var sliderTdownHandler, sliderTmoveHandler;
+          $sliderHandle.bind('touchstart', sliderTdownHandler = function(tsEvt) { // factor code with aboves
+            sliding = true;
+
+            startZooming();
+            $sliderHandle.addClass("active");
+
+            let lastMove = 0;
+            windowBind('touchmove', sliderTmoveHandler = function(tmEvt) {
+              let now = +new Date;
```

Comment:
Are you 100% sure you can't get the time from 'tmEvt'? If there's a timestamp on it, you should use that instead.

Reply:
There actually is timestamp, will change it. I was just using +new Date, as it was used in previous code.